service/src/jamon/org/apache/hive/tmpl/QueryProfileTmpl.jamon

```diff
@@ -57,7 +57,7 @@ private boolean showStats = false;
       </%if>
     </%if>
     <%if showGraph %>
-      <script type="text/javascript" src="/static/js/vis.min.js"></script>
+      <script type="text/javascript" src="/static/js/vis-network.min.js"></script>
```

Comment:
Do we need to include the `vis-network.min.js` file in our source and binary distribution? Can't we simply link to it without copying its source code in this repo?
```
<script type="text/javascript" src="https://unpkg.com/vis-network@9.1.9/standalone/umd/vis-network.min.js"></script>
```

Reply:
I included the file to follow the same manner as earlier.
Yes, we can have it this way as well. I have validated with this way of `src` and it works fine. Please let me know if we can proceed this way.

Reply:
I don't know the history behind the original decision to bundle it inside but for me this approach is better in every aspect.

Reply:
I'm not very clear about license. But the latest change is good to me. :)